src/support/components/FeedbackQuestionsOverlay.tsx

```diff
@@ -28,20 +31,26 @@ import {event} from 'src/cloud/utils/reporting'
 interface OwnProps {
   onClose: () => void
 }
+interface DispatchProps {
+  showOverlay: (arg1: string, arg2: any, any) => {}
+}
+type Props = OwnProps & DispatchProps
 
-const FeedbackQuestionsOverlay: FC<OwnProps> = () => {
+const FeedbackQuestionsOverlay: FC<Props> = props => {
   const {onClose} = useContext(OverlayContext)
   const {id: orgID} = useSelector(getOrg)
   const {id: meID} = useSelector(getMe)
   const [feedbackText, setFeedbackText] = useState('')
 
-  const handleSubmit = () => {
-    // handle form submit
+  const handleSubmit = (e): void => {
+    const {showOverlay} = props
+    e.preventDefault()
```

Comment:
I'd prefer we call this `evt` since `event` is already used in this scope. It's a little bit more descriptive than `e`  (which is commonly used to represent `event` or `error`) and is very handy when searching the whole codebase for errors or events.